### ISSUE TITLE
trouble with CMI /ec and/or opening shulker-boxes directly in inventory

### ISSUE THREAD

**Issue by hitakiller** [JEFF-Media-GbR/Spigot-ChestSort#67]
**Used ChestSort version**
8.7 (latest)

**Used Spigot version**
papermc#318

**Other plugin used**
CMI (latest) 8.6.26.5 ( and many others but this happened with a feature of CMI)
https://www.spigotmc.org/resources/cmi-270-commands-insane-kits-portals-essentials-economy-mysql-sqlite-much-more.3742/

**Describe the bug**
At this point it's just openning searching about this bug, i will complete this issue as i get new information.
With automatic inventory sorting, some shulkers-boxes disapeared. I think it's related with /enderchest and/or opening shulker directly in inventory with shift + rightclicking.
Shulker-boxes are full of many differents items.
I will dig further into this issue, but i have not more information about that.

**To Reproduce**
(to be continued)

**Expected behavior**
(keep shulker-boxes in inventory)

**Additional context**
(to be continued)

**Comment by mfnalex** [comment]
Unfortunately, CMI is a premium plugin and I don't wanna buy it. I will have to ask the author if he can provide me the .jar file for free, however if he does not agree, there is nothing I can do.

**Comment by mfnalex** [comment]
I just saw that I already messaged the author of CMI on May 16th and he didn't reply yet, seems like he isn't interested in getting this fixed. Maybe you should also report the bug at CMI and let me know what answer you got.

**Comment by hitakiller** [comment]
Yes i try to do that, 'cause shulker-boxes disappearing is because of them it happens when chestsort was off.
So thank's for you help, maybe backpack fix will help us with dupplication it seems to be same kind of thing.

Thanks for your help !

**Comment by mfnalex** [comment]
You‘re welcome, please let me know if you get any information from the CMI author

**Comment by mfnalex** [comment]
This seems to be bug in PaperMC in general: https://github.com/PaperMC/Paper/issues/3470
